zebra-utils/src/bin/zebra-checkpoints/main.rs

```diff
@@ -3,8 +3,8 @@
 //! Get all the blocks up to network current tip and print the ones that are
 //! checkpoints according to rules.
 //!
-//! Usage: zebra-checkpoints --cli <cli-path> [-- <zcli-args>...]
-//! `--cli` is the path of the zcash-cli binary as a string.
+//! Usage: zebra-checkpoints [--cli <cli-path>] [-- <zcli-args>...]
```

Comment:
I don't think we should commit the usage to our docs. This is generated automatically by `structopt` so we should probably just direct people to run `zebra-checkpoints --help` to see the usage description.

Reply:
Changed this at https://github.com/ZcashFoundation/zebra/pull/740/commits/c1831c11049373360a2507388e195896e64faa5b